doc/user/content/sql/create-sink.md

```diff
@@ -167,20 +167,21 @@ You can find the topic name for each Kafka sink by querying `mz_kafka_sinks`.
 
 #### Enabling topic reuse after restart
 
-{{< version-added v0.8.2 />}}
+{{< experimental v0.8.2 />}}
 
-This feature is only available in experimental mode. By default, Materialize creates new, distinct topics for sinks after each restart. To enable the reuse of the existing topic instead, when you create a sink, you must:
+By default, Materialize creates new, distinct topics for sinks after each restart. To enable the reuse of the existing topic instead, Materialize must be able to reconstruct the prior history of the sinked object and all objects on which it is dependent--that is, events must have replayable timestamps. This is currently available only for Kafka sources and the views based on them.
+
```

Comment:
@ruchirK does this work?

Reply:
We could mention that Materialize must also "ensure that no duplicates are written to the output topic", which is facilitated by enabling `exactly_once`.

Reply:
this works for me!